static/app/views/alerts/wizard/panelContent.tsx

```diff
@@ -147,4 +147,64 @@ export const AlertWizardPanelContent: Record<AlertType, PanelContent> = {
     ],
     illustration: diagramCrashFreeUsers,
   },
+  app_start_cold: {
+    description: t('App cold start metric track when your app takes too long to launch.'),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  app_start_warm: {
+    description: t(
+      'App warm start metric track when your app takes does not too long to launch.'
+    ),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  frames_slow: {
+    description: t(
+      'Using 60 fps, slow frames are frames that take more time than necessary to render.'
+    ),
+    examples: [
+      t('On iOS, a slow frame occurs when a frame renders after 16.67 ms.'),
```

Comment:
This is only true if the device renders at 60 fps. The iPad Pro, for example, has 120 fps. We should mention that.